cclib/method/nuclear.py

```diff
@@ -100,6 +100,91 @@ def stoichiometry(self):
             sign = "+" if self.data.charge > 0 else "-"
             formula += "(%s%i)" % (sign, magnitude)
         return formula
+    
+    def get_distances(self, atomcoords_index = -1):
```

Comment:
We should just call this `distances` to keep in line with `stiochiometry` in this class and the no-get pattern we have in other places.

Reply:
Yes. I guess that makes sense. I have made changes in the new commit.